src/main/java/emissary/util/shell/Executrix.java

```diff
@@ -748,6 +857,14 @@ public int execute(final String[] cmd, final StringBuilder out, final StringBuil
         return exitValue;
     }
 
+    private void streamData(Process p, byte[] data) throws IOException {
+        try (OutputStream os = new BufferedOutputStream(new DataOutputStream(p.getOutputStream()))) {
+            if (ArrayUtils.isNotEmpty(data)) {
+                os.write(data);
```

Comment:
Will we want to add an os.flush command here? Or will that be automatically handled since this is inside a try-with-object block?

Reply:
I added it